PyDSS/dssInstance.py

```diff
@@ -341,12 +364,18 @@ def DryRunSimulation(self, project, scenario):
 
         return self.ResultContainer.max_num_bytes()
 
+    def initStore(self, hdf_store, Steps, MC_scenario_number=None):
+        if self._Options['Exports']['Result Container'] == 'ResultData':
+            print("initializing store")
+            self.ResultContainer.InitializeDataStore(hdf_store, Steps, MC_scenario_number)
+
     def RunSimulation(self, project, scenario, MC_scenario_number=None):
         startTime = time.time()
         Steps, sTime, eTime = self._dssSolver.SimulationSteps()
         self._Logger.info('Running simulation from {} till {}.'.format(sTime, eTime))
         self._Logger.info('Simulation time step {}.'.format(Steps))
-        if self._Options['Exports']['Result Container'] == 'ResultData' and self.ResultContainer is not None:
+        if self._Options['Exports']['Result Container'] == 'ResultData':
```

Comment:
We don't want to initialize the store if the user has not configured exports, right?

Reply:
For some co-simualtion type simulations run, we do not need the resultcontainer. for NAERM or example i wrote extension to create JSON metadata object at initialization and a new JSON object file very time step. We found this to very effective when working with live dashboards.